packages/driver/src/cy/commands/local_storage.coffee

```diff
@@ -30,17 +30,20 @@ module.exports = (Commands, Cypress, cy, state, config) ->
       null
 
   Commands.addAll({
-    clearLocalStorage: (keys) ->
+    clearLocalStorage: (keys, options = {}) ->
+      _.defaults options, {log: true}
```

Comment:
The signature for this can also be `cy.clearLocalStorage(options)`, so we need to handle that case. Something like:

```coffeescript
if _.isPlainObject(keys)
  options = keys
  keys = null
```

Reply:
Please add a test for this too.

Reply:
Sorry for the late reply!

@chrisbreiding I couldn't understand what you are recommending, `options = keys` did you mean if keys are not present treat the first param as options?
Then maybe we should put a check as well `if (typeof keys is object) then options = keys`
Correct me if I am wrong

Reply:
Yes, that's right. The possible signatures are:

```javascript
cy.clearLocalStorage()
cy.clearLocalStorage(key)
cy.clearLocalStorage(options)
cy.clearLocalStorage(keys, options)
```

Here's an example with `cy.screenshot()`: 

https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/screenshot.coffee#L335

I think it's better to use `_.isPlainObject` though, so it should look like:

```coffeescript
if _.isPlainObject(keys)
  options = keys
  keys = null
```

Reply:
@chrisbreiding Did the changes, can you have a look again and review?